cypress/support/index.js

```diff
@@ -74,3 +74,9 @@ Cypress.on('window:before:load', function (win) {
     set: function () { }
   })
 })
+
+Cypress.on('uncaught:exception', (err, runnable) => {
+  if (err.message.includes('Network request failed')) {
+    return false
+  }
+})
```

Comment:
Dette slår vel til ikkje berre i github actions?

Reply:
Nei det gjør det ikke, men de feila vi får i GA som vi ikke får lokalt er de `Network request failed`

Reply:
Og ut i fra cypress dashboard, så ser jeg at de kalla dette oppstår på er kall til taxonomy som er mocka av oss, så det er noe dumt som skjer et eller annet sted på GA så det er for å stoppe den dumme feilen. For hele testen blir stoppa av noe som ikke burde ha noe å si, siden vi tester ikke taxonomy der dette taxonomy netverk feilen oppstår